test/html.test.js

```diff
@@ -665,3 +672,88 @@ QUnit.test('does not wrap mp4 source buffers', function() {
   );
   QUnit.equal(mediaSource.sourceBuffers.length, 2, 'created native buffers');
 });
+
+QUnit.test('can get activeSourceBuffers', () => {
```

Comment:
I'm cool with fat-arrow functions for test definitions but I think we should be consistent within individual files (and we're using `function` defs up to this point).